custom/icds_reports/views.py

```diff
@@ -137,3 +143,60 @@ def get_tableau_access_token(tableau_user, client_ip):
             return r.text
     else:
         raise TableauTokenException("Token request failed with code {}".format(r.status_code))
+
+
+@location_safe
+@method_decorator([toggles.ICDS_REPORTS.required_decorator(), login_and_domain_required], name='dispatch')
+class DashboardView(TemplateView):
+    template_name = 'icds_reports/dashboard.html'
+
+    @property
+    def domain(self):
+        return self.kwargs['domain']
+
+    @property
+    def couch_user(self):
+        return self.request.couch_user
+
+    def get_context_data(self, **kwargs):
+        kwargs.update(self.kwargs)
+        return super(DashboardView, self).get_context_data(**kwargs)
+
+
+class IcdsDynamicTemplateView(TemplateView):
+
+    def get_template_names(self):
+        return ['icds_reports/icds_app/%s.html' % self.kwargs['template']]
+
+
+@method_decorator([login_and_domain_required], name='dispatch')
+class ProgramSummaryView(View):
+
+    def get(self, request, *args, **kwargs):
+        step = kwargs.get('step')
+
+        # Hardcoded for local tests, in database we have data only for these two days
```

Comment:
what is this here for? it seems like this view does not actually do anything since it is hardcoded.